src/components/GalleryImage.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export default function GalleryImage({ category, id, name, src }) {
+  return (
```

Comment:
@pasternakewa could be removed

Reply:
@vdanylov what exactly?

Reply:
@pasternakewa return, just create arrow function